pkg/keyspace/keyspace.go

```diff
@@ -280,6 +290,83 @@ func (manager *Manager) CreateKeyspace(request *CreateKeyspaceRequest) (*keyspac
 	return keyspace, nil
 }
 
+// CreateKeyspaceByID create a keyspace meta with given config and save it to storage.
+// This function is used by next-gen TiDB.
```

Comment:
Better not to refer this directly in our codebase IMO.

Reply:
ok, removed